lib/cloudinary/utils.rb

```diff
@@ -839,10 +839,8 @@ def self.resource_type_for_format(format)
     case
     when self.supported_format?(format, IMAGE_FORMATS)
       'image'
-    when self.supported_format?(format, VIDEO_FORMATS)
+    when self.supported_format?(format, VIDEO_FORMATS + AUDIO_FORMATS)
```

Comment:
`VIDEO_FORMATS + AUDIO_FORMATS` will cause unneeded memory allocation of every function call, replace with:

`when self.supported_format?(format, VIDEO_FORMATS), self.supported_format?(format, AUDIO_FORMATS)`

or create another variable at the top of the file `AUDIO_VIDEO_FORMATS = VIDEO_FORMATS + AUDIO_FORMATS`